bionic/protocols.py

```diff
@@ -198,14 +198,22 @@ class PicklableProtocol(BaseProtocol):
     """
     Decorator indicating that an entity's values can be serialized using the
     ``pickle`` library.
+
+    Parameters:
+        protocol: int (default: 4)
```

Comment:
I don't think this name will break anything, but it may be a little bit confusing. WDYT about something like `pickle_protocol_version`?